### PR TITLE
Update zh_cn.json

### PR DESCRIPTION
**Pull request by Neubulae**
sorry for the bad format

### CODE REVIEW

**Comment by BloCamLimb** [comment]
it's worse than before, certainly unusual

**Comment by Neubulae** [comment]
> it's worse than before, certainly unusual

😢
could you give specific reasons for what's bad

**Comment by Snownee** [comment]
> it's worse than before, certainly unusual

Really? I think this version of translation correctly conveys the meme-ness of the original translation. Instead of being unnecessarily formal at most of the time, this colloquial approach can be a bit more appealing. Anyway, this version of translation looks fine to me, this is just my opinion.

**Comment by LeoCTH** [comment]
looks good

(its customary to have funni translations for advancements lol)

File: src/main/resources/assets/indrev/lang/zh_cn.json [hunk 1]
```diff
@@ -180,255 +180,255 @@
...
   "gui.indrev.tooltip.maxInput": "最大输入：",
   "gui.indrev.tooltip.maxOutput": "最大输出：",
   "gui.indrev.tooltip.maxEnergyStored": "最大储能：",
   "gui.indrev.tooltip.energyCost": "能耗：",
-  "gui.indrev.tooltip.processSpeed": "速度：",
+  "gui.indrev.tooltip.processSpeed": "速率：",
   "gui.indrev.tooltip.ratio": "发电：",
   "gui.indrev.tooltip.press_shift": "按 Shift 查看详细信息",
-  "gui.indrev.tooltip.lftick": "%s LF/tick",
+  "gui.indrev.tooltip.lf刻": "%s LF/刻",
```
Comment (LeoCTH) [inline src/main/resources/assets/indrev/lang/zh_cn.json:207]: this is concarn plz fix

**Review by LeoCTH — CHANGES_REQUESTED** [review]

File: src/main/resources/assets/indrev/lang/zh_cn.json (hunk 1 above)
Comment (Neubulae) [reply src/main/resources/assets/indrev/lang/zh_cn.json:207]: fixx

Comment (Neubulae) [reply src/main/resources/assets/indrev/lang/zh_cn.json:207]: https://github.com/GabrielOlvH/Industrial-Revolution/pull/58/commits/807be087c82c3ea7ccefb26d3931e8bb27e7416a
here, github thank :microthonk:

Comment (LeoCTH) [reply src/main/resources/assets/indrev/lang/zh_cn.json:207]: ~~why is it not on the pr lolz~~

**Comment by GabrielOlvH** [comment]
Thanks for the translations!

**Pull request merged**